plotly_resampler/figure_resampler.py

```diff
@@ -417,6 +417,34 @@ def to_same_tz(
 
         return hf_series[to_same_tz(t_start) : to_same_tz(t_stop)]
 
+    @property
+    def hf_data(self):
+        return list(self._hf_data.values())
```

Comment:
Calling `.values()` on a dict does not return a fixed order, right? Could this be a problem?

Reply:
Oh wow, TIL "Dicts preserve insertion order in Python 3.7+"
https://stackoverflow.com/a/613218/12180712